stdlib/distutils/sysconfig.pyi

```diff
@@ -10,8 +12,15 @@ project_base: str
 python_build: bool
 
 def expand_makefile_vars(s: str, vars: Mapping[str, str]) -> str: ...
+@overload
+@deprecated("SO is deprecated, use EXT_SUFFIX")
+def get_config_var(name: Literal["SO"]) -> int | str | None: ...
+@overload
 def get_config_var(name: str) -> int | str | None: ...
-def get_config_vars(*args: str) -> Mapping[str, int | str]: ...
+@overload
+def get_config_vars() -> dict[str, str | int]: ...  # type: ignore[overload-overlap]
```

Comment:
You can probably get rid of this ignore by making the other overload have an extra positional-only argument, like `def get_config_vars(arg: str, /, *args: str)`.

Reply:
Alright so same as my comment above, I'll do that then, thanks for your input